Maryam-Marjoori/homework_3/4-fizzbuzz-homework/app.js

```diff
@@ -0,0 +1,76 @@
+
+// Relying on your newfound knowledge of loops, combine loops and if/else statements together and incrementally build the common fizzbuzz loop.
+
+// - In the loop, every time a number is divisible by **3**, instead of logging the number itself, the word "fizz" should appear.
+// - If the number is divisible by  **5**, the word "buzz" should be logged.
+// - If the number is divisible by both **3** and  **5**, then the word "fizzbuzz" should be logged.
+
+// Follow the steps below.
+
+// ##### Step 1:
+// Construct a for loop that iterates through, and `console.log()`'s out, numbers 1 - 100.
+
+
+// ##### Step 2:
+// Add an `if/else` statement that logs the string `"fizz"` if the value being iterated over is divisible by `3`; otherwise, log out the value.
+
+
+
+// ##### Step 3:
+// Add an `else if` clause that logs the string `"buzz"` if the value being iterated over is divisible by `5`.
+
+
+
+// ##### Step 4:
+// Add an additional `else if` clause that logs the string `"fizzbuzz"` if the value being iterated over is divisible by both `3` and `5`.
+
+
+// ************************************************************ First Approach ************************************************************
+
+for ( var totalNum = 100; totalNum > 0 ; totalNum-- ){
+    if ( (totalNum % 3) === 0 ){
+        console.log("fizz");
+    }
+    if ( (totalNum % 5) === 0 ){
+        console.log("buzz");
+    }
+    if (totalNum % 5 === 0 && totalNum % 3 === 0){
+        console.log("Fizz Buzz");
+    }
```

Comment:
The `totalNum % 3` and `totalNum % 5` conditions will apply before the last `if` conditional is hit, so it's best practice to place more specific conditions first, and less specific conditions fall closer to the bottom.